docs/jwt.md

```diff
@@ -36,28 +36,15 @@ As with other login types, there are additional fields (e.g. `device_id` and
 
 ## Preparing Synapse
 
-The JSON Web Token integration in Synapse uses the
-[`PyJWT`](https://pypi.org/project/pyjwt/) library, which must be installed
-as follows:
-
- * The relevant libraries are included in the Docker images and Debian packages
-   provided by `matrix.org` so no further action is needed.
-
- * If you installed Synapse into a virtualenv, run `/path/to/env/bin/pip
-   install synapse[pyjwt]` to install the necessary dependencies.
-
- * For other installation mechanisms, see the documentation provided by the
-   maintainer.
-
-To enable the JSON web token integration, you should then add an `jwt_config` section
```

Comment:
As discussed below, server owners will now need the `authlib` package instead of `pyjwt` to support authlib.

Can you restore this and make the following changes:
- refer to `authlib` rather than `PyJWT`
- change `pip install synapse[pyjwt]` to `pip install synapse[jwt]` (as it should have been originally!)
  - Note that the thing in square brackets is the name of the extra feature, not the name of a package.

Reply:
Just saw, that the changed URL in `config/jwt.py` collides with #12941 so I merged and fixed this locally.

Then I saw that each PR has _one_ commit ... so you're squashing before a merge ... which means I'll rebase all my commits.

Is that correct?

Reply:
> Then I saw that each PR has one commit ... so you're squashing before a merge ... which means I'll rebase all my commits.
>
> Is that correct?

Assuming you merged in the `develop` branch, you should be okay to push the merge commit to this branch. (Where possible we prefer to avoid rebases on PRs since it makes reviewing more difficult.)